src/main/java/core/basesyntax/model/RegistrationServiceImpl.java

```diff
@@ -0,0 +1,40 @@
+package core.basesyntax.model;
+
+import core.basesyntax.dao.StorageDao;
+import core.basesyntax.dao.StorageDaoImpl;
+import core.basesyntax.exception.RegistrationException;
+import core.basesyntax.service.RegistrationService;
+
+public class RegistrationServiceImpl implements RegistrationService {
+    static final int AGE_PERMISION = 18;
+    static final int MIN_PASSWORD_LENGTH = 5;
```

Comment:
add constant for login length too (better option), or change the naming and have one for both